deployer/auth.py

```diff
@@ -63,6 +65,15 @@ def _get_callback_url_list(self, domains):
 
         return [f'https://{domains}/hub/oauth_callback']
 
+    def _get_allowed_logout_url_list(self, domains):
+        if isinstance(domains, list):
+            logout_urls = []
```

Comment:
Since this is a farily simple for loop, can we make it a list comprehension instead?